x-pack/test/security_solution_cypress/cypress/tasks/api_calls/exceptions.ts

```diff
@@ -47,7 +47,7 @@ export const createExceptionList = (
 
 export const createExceptionListItem = (
   exceptionListId: string,
-  exceptionListItem?: ExceptionListItem
+  exceptionListItem?: Partial<ExceptionListItem>
```

Comment:
Just one question here. 
 In TS, you don't need to actually set the parameter `exceptionListItem` as optional, because you are using `Partial`.  Is this correct?

Reply:
No, it we remove optional it will expect at least empty object

https://www.typescriptlang.org/play/?#code/GYVwdgxgLglg9mABAQwBQAdkCdkFsDOAXIgArazIA2APAN4BQiiYeApsflFjGAOYDcjFL3bMQuAEassggL4A+AJS1Z9emgZMWuUQCIAjACYAzLoA09WYrVpFQA

Reply:
Wow!  I'm so glad I asked!  Thanks a lot @nkhristinin !